src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+---
+title: A Developer's Guide to Ethereum, Pt. 2
+description: Introduction to Ethereum with web3.py and Python.
+icons: ["ethereum"]
+---
+
+import { LessonHeader } from "../../../components/mdx/LessonHeader";
+import Layout from "../../../components/Layout";
+
+<Layout
+    title="A Developer's Guide to Ethereum, Pt. 2" 
+    description="Introduction to Ethereum with web3.py and Python." 
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 2" />
+
+Welcome back! In the first installment of this series we covered a lot of ground
```

Comment:
We could do with a header here for this first paragraph. Other than that, the use of ## headers and ### sub-headers are really meaningfully structured and therefore helpful for @uma-l and @markkos and dev team.